quiz.js

```diff
@@ -37,9 +50,23 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.update({
+  name: 'Stark' }, { $push: {person: { name: 'Ned'}}});
```

Comment:
`person` is probably not the best name for this property, since it's going to hold a list of all people associated with that particular house.